src/crypto/crypto_keys.cc

```diff
@@ -892,30 +892,44 @@ size_t KeyObjectData::GetSymmetricKeySize() const {
   return symmetric_key_.size();
 }
 
+bool KeyObjectHandle::HasInstance(Environment* env,
+                                  v8::Local<v8::Value> value) {
+  return !value.IsEmpty() && value->IsObject() &&
+         GetConstructorTemplate(env)->HasInstance(value);
```

Comment:
GetConstructorTemplate leaks handles into the parent handlescope when it lazy-creates the template. Maybe change this to something like:
```c++
auto t = env->keyobjecthandle_ctor_template();
return !t.IsEmpty() && t->HasInstance(value);
```
And remove GetConstructorTemplate again, it's a bit of a footgun.

The IsObject() check isn't really necessary. Doesn't hurt either but HasInstance() checks that anyway.

Reply:
Handled in #45912